org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/DebriefLiteApp.java

```diff
@@ -20,6 +20,7 @@
 import java.awt.Component;
 import java.awt.Cursor;
 import java.awt.Dimension;
+import java.awt.FlowLayout;
```

Comment:
@saulhidalgoaular - could you put the Dialog creation/management into another class?

DebriefLiteApp already carries too many concepts, IMHO.